test/functional/page_objects/unified_search_page.ts

```diff
@@ -50,4 +42,10 @@ export class UnifiedSearchPageObject extends FtrService {
       `[data-test-subj="text-based-languages-switcher"] [title="${language}"]`
     );
   }
+
+  public async switchToDataViewMode() {
+    await this.testSubjects.click('esql-menu-button');
+    await this.testSubjects.click('switch-to-dataviews');
+    await this.testSubjects.click('unifiedSearch_switch_noSave');
```

Comment:
Same as above, this is a helper function. Assertions happen where this reusable function is being used.

Reply:
Agree with @wayneseymour. It would be better to wait for `switch-to-dataviews` button to appear before clicking on it. Same for `unifiedSearch_switch_noSave` button.